The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ois-1.3.0.orig/demos/Makefile.am
+++ ois-1.3.0/demos/Makefile.am
@@ -4,8 +4,8 @@ noinst_PROGRAMS = ConsoleApp FFConsoleTe
 
 ConsoleApp_SOURCES = OISConsole.cpp
 ConsoleApp_LDFLAGS = -L$(top_builddir)/src
-ConsoleApp_LDADD = -lOIS
+ConsoleApp_LDADD = -lOIS -lX11
 
 FFConsoleTest_SOURCES = FFConsoleDemo.cpp
 FFConsoleTest_LDFLAGS = -L$(top_builddir)/src
-FFConsoleTest_LDADD = -lOIS
+FFConsoleTest_LDADD = -lOIS -lX11
